### ISSUE TITLE
Center of Excellence (CoE) 3. Turn on: CLEANUP - Admin | Sync Template v3 (Check Deleted) dont work

### ISSUE THREAD

**Issue by Maxiiii-s** [microsoft/coe-starter-kit#891]
Hello community,

I am in the process of setting up the Centre of Excellence. However, I am failing at point 3 "Turn on: CLEANUP - Admin | Sync Template v3 (Check Deleted)"

The flow could not be activated: CLEANUP - Admin | Sync Template v3 (Check Deleted). Error: Error saving the flow. Code: InvalidPaginationPolicy, message: 'The pagination policy of the workflow execution action "Get_Environments" of type "OpenApiConnection" in row 1 and column 26222 is invalid. The value specified for the property "minimumItemsCount" exceeds the maximum value allowed. Actual value: 100000. Maximum value: 5000.".

What can I do now?

Have I done something wrong?

Thank you very much for your response in advance.

Many greetings

Maxiiii-s

**Comment by Maxiiii-s** [comment]
Thanks mikefactorial, is there anyone who has the same problem too?

**Comment by rpothin** [comment]
Hey,

Yes, I got the same issue deploying the **CenterofExcellenceCoreComponents_2.2.20210707.3_managed** solution few days ago.
As a temporary workaround, I reduced **Threshold** setting for all Dataverse list records actions from 100000 to 5000.

I got the same error for other flows in the solution too.

Information about my environment:
- Type: Production
- Region: Europe
- Database version: 9.2.21064.00144

I hope it will help find a solution to this issue.

Have a great day.
